plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ArrowFunction_Part2_Transformation.xtend

```diff
@@ -12,14 +12,17 @@ package org.eclipse.n4js.transpiler.es.transform
 
 import org.eclipse.n4js.n4JS.ArrowFunction
 import org.eclipse.n4js.transpiler.Transformation
+import org.eclipse.n4js.transpiler.TransformationDependency.Optional
 import org.eclipse.n4js.transpiler.TransformationDependency.Requires
 
 import static org.eclipse.n4js.transpiler.TranspilerBuilderBlocks.*
 
 import static extension org.eclipse.n4js.typesystem.RuleEnvironmentExtensions.*
 
 /**
+ * Part 2 of {@link ArrowFunction_Part1_Transformation}.
  */
+@Optional(ES5plus)
```

Comment:
good find! `ES5Plus` is wrong here.
Changed to `ArrowFunctions` (also in other places)